packages/database/src/migrations/20210202061216-AddFetpGraduateIndividualProfiles-modifies-data.js

```diff
@@ -0,0 +1,209 @@
+'use strict';
+
+import { arrayToDbString, insertObject } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const dataServices = [{ isDataRegional: true }];
+
+const buildTable = report => {
+  const mapElementsToCells = element =>
+    report.elementConfigs && element in report.elementConfigs
+      ? [report.elementConfigs[element]]
+      : [element];
+
+  const dataBuilderConfig = {
+    rows: Object.values(report.elements),
+    cells: Object.keys(report.elements).map(mapElementsToCells),
+    columns: ['main'],
+    entityAggregation: {
+      aggregationType: 'RAW',
+      dataSourceEntityType: ['individual'],
+      aggregationEntityType: 'individual',
+    },
```

Comment:
Not sure if this is required, since it maps the aggregation source type to itself? I'm not fully across `entityAggregation`, so the question is mainly for my benefit 😄

Reply:
You are correct need `dataSourceEntityType` but not `aggregationEntityType` in this case

Reply:
Good to know! Not a big deal but I guess the simpler the better